tests/ryzenai/test_modeling.py

```diff
@@ -287,3 +298,90 @@ def test_model(self, model_id):
         self.assertEqual(baseline_ops["dpu"], current_ops["dpu"], f"DPU operators do not match! {current_ops}")
 
         gc.collect()
+
+
+class RyzenAIModelForCausalLMIntegrationTest(unittest.TestCase, RyzenAITestCaseMixin):
+    SUPPORTED_ARCHITECTURES = {
+        "opt",
+        "llama",
+        "mistral",
```

Comment:
gpt bigcode not here?

Reply:
Added gpt bigcode, there are other models also which should be supported without a change. Would add them in other PR after full testing.